Formula/man-db.rb

```diff
@@ -90,12 +90,12 @@ def caveats
     ENV["PAGER"] = "cat"
     if OS.mac?
       output = shell_output("#{bin}/gman true")
-      assert_match "BSD General Commands Manual", output
+      assert_match "General Commands Manual", output
       assert_match(/The true utility always returns with (an )?exit code (of )?zero/, output)
     else
       output = shell_output("#{bin}/gman gman")
       assert_match "gman - an interface to the system reference manuals", output
-      assert_match "https://savannah.nongnu.org/bugs/?group=man-db", output
```

Comment:
This should be an upstream issue since the updated url is not working, but I am updating to satisfy the test.

Reply:
It looks like your test is expecting a hyphen-minus U+002D but getting a hyphen U+2010.

This is because man-db's man(1) page does not escape the hyphens in its URLs, which appear in the document as ordinary formatted text (as of this writing).

https://gitlab.com/man-db/man-db/-/blob/main/man/man1/man.man1#L1300

Per its NEWS file, groff 1.23.0 no longer conceals such errors in man pages by default.

https://git.savannah.gnu.org/cgit/groff.git/tree/NEWS?h=1.23.0#n206

What is it that these tests are trying to validate?  It looks like they might be trying simply to check that the text of the page was not truncated.  A comment in the test source might help.

In any case, you can run man-db man ("gman") with the option `-T ascii` to stop the rendering of hyphens from confounding your test.  If you do that you will want to also update the expectations of your tests to match ASCII hyphens, U+002D.

Reply:
> What is it that these tests are trying to validate? It looks like they might be trying simply to check that the text of the page was not truncated. A comment in the test source might help.

yeah, I think it is just do the test url check. I can add the ascii option.

Reply:
it does not work with `-T ascii`

```
==> /home/linuxbrew/.linuxbrew/Cellar/man-db/2.11.2/bin/gman -T ascii gman
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/ignorable.rb:29:in `block in raise'
Minitest::Assertion: Expected /https:\/\/savannah\.nongnu\.org\/bugs\/\?group=man\-db/ to match "%!PS-Adobe-3.0\n%%Creator: groff version 1.23.0\n%%CreationDate: Sun Jul 16 23:51:42 2023\n%%DocumentNeededResources: font Times-Italic\n%%+ font Times-Roman\n%%+ font Times-Bold\n%%
```

I am gonna just skip the link test, it is not necessary for homebrew to have to have that.